src/components/home-page/hero-section/component/Headline.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Carousel } from 'react-responsive-carousel';
+
+function Headline() {
+    return (
+        <>
+            <div className="align-center w-full flex justify-evenly gap-4 text-center md:hidden">
+                <Carousel showArrows={false} showStatus={false} infiniteLoop={true} className="w-full">
```

Comment:
Is there any way we can customize the color of the indicators to match the ones used in the Figma?

![image](https://user-images.githubusercontent.com/14112766/177626255-9b5da2cf-98b4-43e4-843a-48f937789f69.png)